### ISSUE TITLE
move item package to world state

### ISSUE THREAD

**Issue by Planutim** [skycoin/cx-game#523]

**Comment by ted537** [comment]
What does this mean? Should the world package contain the item package?

**Comment by Planutim** [comment]
yes, synth was mentioning. Keeping the data in a separate package might be bad, because there will be multiple worlds
